charts/refinery/values.yaml

```diff
@@ -255,6 +255,13 @@ tolerations: []
 
 affinity: {}
 
+podDisruptionBudget:
```

Comment:
Should this have a comment documenting it?

Reply:
Agreed.  Please add a comment above `podDisruptionBudget` describing what this section enables and when someone might what to enable it.

Reply:
Done